Tests/client-sdk-swiftTests/client_sdk_swiftTests.swift

```diff
@@ -55,4 +55,91 @@ final class client_sdk_swiftTests: XCTestCase {
             "Unexpected response: \((pubResp as! TopicPublishError).description)"
         )
     }    
+    
+    func testTopicClientSubscribes() async throws {
+        let creds = try CredentialProvider.fromEnvironmentVariable(envVariableName: "MOMENTO_API_KEY")
+        let client = TopicClient(configuration: Default.latest(), credentialProvider: creds)
+        XCTAssertNotNil(client)
+        
+        let invalidCacheNameResp = await client.subscribe(
+            cacheName: "",
+            topicName: "test-topic"
+        )
+        XCTAssertTrue(
+            invalidCacheNameResp is client_sdk_swift.TopicSubscribeError,
+            "Unexpected response: \(invalidCacheNameResp)"
+        )
+        let invalidCacheNameErrorCode = (invalidCacheNameResp as! TopicSubscribeError).errorCode
+        XCTAssertEqual(
+            invalidCacheNameErrorCode, MomentoErrorCode.INVALID_ARGUMENT_ERROR,
+            "Unexpected error code: \(invalidCacheNameErrorCode)"
+        )
+        
+        let invalidTopicNameResp = await client.subscribe(
+            cacheName: "test-cache",
+            topicName: ""
+        )
+        XCTAssertTrue(
+            invalidTopicNameResp is client_sdk_swift.TopicSubscribeError,
+            "Unexpected response: \(invalidTopicNameResp)"
+        )
+        let invalidTopicNameErrorCode = (invalidTopicNameResp as! TopicSubscribeError).errorCode
+        XCTAssertEqual(
+            invalidTopicNameErrorCode, MomentoErrorCode.INVALID_ARGUMENT_ERROR,
+            "Unexpected error code: \(invalidTopicNameErrorCode)"
+        )
+        
+        let subResp = await client.subscribe(
+            cacheName: "test-cache",
+            topicName: "test-topic"
+        )
+        XCTAssertTrue(
+            subResp is TopicSubscribeSuccess,
+            "Unexpected response: \((subResp as! TopicSubscribeError).description)"
+        )
+    }
+    
+    func testTopicClientPublishesAndSubscribes() async throws {
```

Comment:
This test is still a little flaky sometimes but I'm not sure why. Sometimes it passes, other times it receives only heartbeats.